lib/rmt/scc.rb

```diff
@@ -129,7 +128,7 @@ def create_product(item, root_product_id = nil, base_product = nil, recommended
       ProductsExtensionsAssociation.where(root_product_id: root_product_id).destroy_all
     end
 
-    item[:extensions].each do |ext_item|
+    item[:extensions]&.each do |ext_item|
```

Comment:
Why wouldn't a product item have `:extensions` key? It's the same serializer for everything.

Reply:
spec issue. fixed